src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -481,13 +483,30 @@ protected function getAddColumnInstructions(Table $table, Column $column)
             $this->getColumnSqlDefinition($column)
         );
 
-        if ($column->getAfter()) {
-            $alter .= ' AFTER ' . $this->quoteColumnName($column->getAfter());
-        }
+        $alter .= $this->afterClause($column);
 
         return new AlterInstructions([$alter]);
     }
 
+    /**
+     * Exposes the MySQL syntax to arrange a column `FIRST`.
+     *
+     * @param Column $column The column being altered.
+     *
+     * @return string The appropriate SQL fragment.
+     */
+    protected function afterClause(Column $column)
+    {
+        $after = $column->getAfter();
+        if (empty($after)) {
+            return '';
+        } elseif ($after === self::FIRST) {
```

Comment:
no else needed on early returns.

Reply:
I have no idea what you're talking about. 😝